protx-client/src/components/ProTx/components/charts/AnalyticsStateDistribution.jsx

```diff
@@ -4,32 +4,25 @@ import PropTypes from 'prop-types';
 import { LoadingSpinner } from '_common';
 import MainPlot from './MainPlot';
 import ChartInstructions from "./ChartInstructions";
+import './PlotDetails.css';
 
 
-function AnalyticsStateDistribution({geography}) {
+function AnalyticsStateDistribution({geography, analyticsType}) {
```

Comment:
suggestion: add `analyticsType` to the propTypes.

[ we don't have any client-side linting for these things at the moment. we should tackle that next sprint]